frontend/sw360-portlet/src/main/webapp/js/main.js

```diff
@@ -263,11 +263,22 @@ function stringToBoolean(s) {
     return s == "true";
 }
 
+function truncateText(text, length) {
```

Comment:
why not use `overflow: ellipsis`?

Reply:
removed function `truncateText`

added:
`
            function renderDescription(description, type, row) {
                return $('<span>', {
                    title: description,
                    "class": "sw360-ellipsis",
                }).text(description);
            }
`